src/screens/AddSiteFormScreen.js

```diff
@@ -465,6 +489,10 @@ const styles = {
         display: 'flex',
         flexDirection: 'row',
         justifyContent: 'space-around'
+    },
+    pickerStyle: {
+        width: 250,
+        alignSelf: 'center'
```

Comment:
Picker is now only `250` wide instead of taking up the entire width of the form. This make scrolling through the form much easier.